### PR TITLE
adding PVC and fixing TOC table

### PR DESCRIPTION
**Pull request by csantanapr**
Can't "PVC" in search, also the TOC (Table of Content) was empty the flags where not showing up.

**Before**
image

**After**
image

### CODE REVIEW

File: docs/serving/configuration/feature-flags.md [hunk 1]
```diff
@@ -78,8 +78,6 @@ Beta stage
 GA stage
 : The feature is allowed by default.
 
-# Available Flags
```
Comment (snneji) [inline docs/serving/configuration/feature-flags.md:81]: Maybe we should keep this heading but change it to an H2. The heading provides separation between the information about how flags are configured, and the flags you can use. 

The "Available flags" heading should be an H2 header and the flags such as "Multiple containers" under it should be H3 headings.

**Review by snneji — DISMISSED** [review]

File: docs/serving/configuration/feature-flags.md (hunk 1 above)
Comment (csantanapr) [reply docs/serving/configuration/feature-flags.md:81]: made the suggested change

**Comment by csantanapr** [comment]
this is how it looks now using H3
@snneji PTAL

**Comment by csantanapr** [comment]
image

**Review by snneji — COMMENTED** [review]
LGTM, but one small issue - should the final commit be in another PR? There is a change to the `extra.css` file on this PR that maybe shouldn't be here.

**Comment by csantanapr** [comment]
Yes let remove the CSS file 👍

**Comment by csantanapr** [comment]
@snneji I removed the exta.css file, PTAL

**Pull request merged**